### DISCUSSION TITLE
Audit Node Module folder with YARA rules

### DISCUSSION THREAD

**Discussion by rpgeeganage** [npm/feedback#307] [General]
**(New rules, PRs, feedbacks are highly appreciated)**

GitHub Repo: [https://github.com/rpgeeganage/audit-node-modules-with-yara](https://github.com/rpgeeganage/audit-node-modules-with-yara)

**Help to detect supplier chain attacks**

**About**:
- The purpose of this tool is to run a given set of YARA rules against the given `node_module` folder.
- With this approach, We can define YARA rules to identify suspicious scripts which are injected into node packages.
- This package can be added to the CI/CD pipeline

**Comment by ljharb** [comment]
This repo is for providing feedback on npm itself; are you looking for feedback on your npm package?

**Reply by rpgeeganage** [reply]
@ljharb Sorry for posting in the wrong place, I'm looking for feedback for my package.
